code/__DEFINES/reagents.dm

```diff
@@ -104,18 +104,18 @@
 ////Used to force an equlibrium to end a reaction in reaction_step() (i.e. in a reaction_step() proc return END_REACTION to end it)
 #define END_REACTION "end_reaction"
 
-///Minimum requirement for addiction buzz to be met
-#define MIN_ADDICTION_REAGENT_AMOUNT 2
+///Minimum requirement for addiction buzz to be met. Addiction code only checks this once every two seconds, so this should generally be low
+#define MIN_ADDICTION_REAGENT_AMOUNT 1
 ///Nicotine requires much less in your system to be happy
 #define MIN_NICOTINE_ADDICTION_REAGENT_AMOUNT 0.1
 #define MAX_ADDICTION_POINTS 1000
 
 ///Addiction start/ends
-#define WITHDRAWAL_STAGE1_START_CYCLE 1
-#define WITHDRAWAL_STAGE1_END_CYCLE 60
-#define WITHDRAWAL_STAGE2_START_CYCLE 61
-#define WITHDRAWAL_STAGE2_END_CYCLE 120
-#define WITHDRAWAL_STAGE3_START_CYCLE 121
+#define WITHDRAWAL_STAGE1_START_CYCLE 120
```

Comment:
This is far too long though. Lower this.

Reply:
I changed it to 1 minute. Not willing to go below this, since you can't microdose anymore and I don't want people to have to focus their entire round on only taking drugs